src/pages/Transactions/transactionUtils.tsx

```diff
@@ -13,16 +13,28 @@ const pricePerValidator = new BigNumber(PRICE_PER_VALIDATOR);
 const TX_VALUE = pricePerValidator.multipliedBy(1e18).toNumber();
 
 const isUserRejectionError = (error: any) => {
-  if (error.code === 4001) return true; // metamask
+  if (error.code === 4001) return true; // metamask reject
+  if (error.code === -32603)
+    if (
+      error.message.includes(
+        'MetaMask Tx Signature: User denied transaction signature.'
+      )
+    )
+      // do nothing because there are different cases with the same error-code ;
+      return true; // metamask reject
   if (
-    error.message ===
-    'MetaMask Tx Signature: User denied transaction signature.'
+    error.message.includes(
+      'Ledger device: Condition of use not satisfied (denied by the user?)'
+    ) &&
+    error.code === -32603
   )
-    return true; // metamask
-  if (error.message === 'User denied transaction signature.') return true; // portis
+    return true; // ledger reject via metamask
```

Comment:
It looks like it could be refactored the `error.code === -32603` a bit
```js
  if (error.code === -32603)
    // do nothing because there are different cases with the same error-code ;
    if (
        // (1) metamask reject
        error.message.includes(
        'MetaMask Tx Signature: User denied transaction signature.'
        ) || (
        // (2) Ledger reject via metamask
        error.message.includes(
        'Ledger device: Condition of use not satisfied (denied by the user?)'
        ))) {
        return true;
    }
```

Does `Fortmatic RPC Error: [-32603] Fortmatic: User denied transaction.` case also has `error.code === -32603`?